### PR TITLE
After Pulsar Virtual Summit Europe 2023

### PR DESCRIPTION
**Pull request by visortelle**
This PR adds doc for #xyz

This PR fixes #xyz 

- [ ] `doc` 
- [ ] `doc-required` 
- [x] `doc-not-needed` 
- [ ] `doc-complete` 

- Removed the announcement bar.
- Added Pulsar Summit Europe 2023 to the events list.
- Changed Featured Event block text.

Screenshot 2023-05-24 at 9 41 42 AM

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -142,16 +142,16 @@ module.exports = {
...
-      id: "summit",
-      content: renderAnnouncementBar(
-        "Pulsar Virtual Summit Europe 2023",
-        "https://pulsar-summit.org/event/europe-2023"
-      ),
-      backgroundColor: "#282826",
-      textColor: "#fff",
-      isCloseable: false,
-    },
+    // announcementBar: {
```
Comment (tisonkun) [inline docusaurus.config.js:145]: thought - perhaps we can develop a default announcementBar later.

**Review by tisonkun — APPROVED** [review]
Thanks for taking care of this!

As the event has ended, I agree we update the metadata.

File: docusaurus.config.js (hunk 1 above)
Comment (visortelle) [reply docusaurus.config.js:145]: If you have a concrete suggestion of how it should work, I can implement it.

Comment (tisonkun) [reply docusaurus.config.js:145]: I don't know, lol.

Other projects may set a banner for "star me", but I don't think it's a good idea.

**Pull request merged**